app/services/stream-info.ts

```diff
@@ -44,7 +44,7 @@ export class StreamInfoService extends StatefulService<IStreamInfoServiceState>
   streamInfoChanged = new BehaviorSubject<IStreamInfo>(StreamInfoService.initialState);
 
   init() {
-    this.refreshStreamInfo();
+    this.refreshStreamInfo().catch(e => null);
```

Comment:
why are we swallowing the error here but you changed the promise rejection from null to a text string?

Reply:
Because `refreshStreamInfo()` is calling not only from this place. The rejection message helps to find unhandled promises rejections